### PR TITLE
F/class list

### PR DESCRIPTION
**Pull request by fanglinfang**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ after_success:
         "develop")
           export APP_INSTANCE="dev";
           ;;
-        "feature/vue")
+        "f/class-list")
```
Comment (SarthakSingh31) [inline .travis.yml:57]: This probably should be changed back to `feature/vue`

File: myuw/static/vue/components/classlist/class-student-list.vue [hunk 2]
```diff
@@ -0,0 +1,135 @@
+<template>
+  <uw-card v-if="showCard"
+           :loaded="isReady"
+           :errored="isErrored"
+           :errored-show="isErrored"
```
Comment (SarthakSingh31) [inline myuw/static/vue/components/classlist/class-student-list.vue:5]: This is not required as errored-show is true by default. The error will not be shown is errored is false.

File: .travis.yml (hunk 1 above)
Comment (fanglinfang) [reply .travis.yml]: Definitely, updated.

File: myuw/static/vue/components/classlist/class-student-list.vue (hunk 2 above)
Comment (fanglinfang) [reply myuw/static/vue/components/classlist/class-student-list.vue]: Thanks, updated!

**Comment by fanglinfang** [comment]
/app/myuw/static/vue/components/classlist/photo-list.vue

  13:1  error  This line has a length of 91. Maximum allowed is 80  max-len

**Pull request closed without merging**